docs/index.md

```diff
@@ -1,23 +1,5 @@
 ---
-
+title: See Main Page
```

Comment:
This isn't a site page so not needed. Good change.

Reply:
Thanks!!